packages/cli/src/Init.ts

```diff
@@ -35,6 +35,16 @@ datasource db {
 `
 }
 
+export const withModelSchema = (provider: ConnectorType = 'postgresql') => {
+  return defaultSchema(provider).concat(`
+model User {
+  id    Int     @id @default(autoincrement())
+  email String  @unique
+  name  String?
+}
```

Comment:
Hey thanks for your contribution! I think this will produce an invalid schema for `mongodb` and `cockroachdb` providers, so we would need to add that and test that so we don't accidentally suggest an invalid schema now and in the future if we add a new provider.

Reply:
That would only matter if the provider is specified as well, otherwise it is `postgresql` by default and it's fine then.

Reply:
Hello ! You're right, I just checked online and on your documentation and it will produce an incorrect schema for both the provider you mentioned.

I'm not sure about how I should do that, do you have any idea on how I should tackle this problem ? 

Maybe I could create a new file in the "utils" folder dedicated to the creation of schemas ? 

Thanks for your feedback 😊

Reply:
I think you can use the `provider` parameter that is provided to `withModelSchema` already to include a _different_ model for MongoDB and CockroachDB that is valid for these.